__tests__/app.test.js

```diff
@@ -150,6 +150,50 @@ describe("/api/reviews/:review_id/comments", ()=> {
     expect(res.body).toEqual({msg:"Error - review ID not found"})
 })
 })
+test("POST - status 201 - responds with the added comment object", () => {
+  return request(app)
+  .post("/api/reviews/8/comments")
+  .expect(201)
+  .send({username:'efie', body:'great game'})
+  .then((result => {
+    
+    const {comment}= result.body
+    expect(comment.body).toBe('great game')
+    expect(comment.author).toBe("mallionaire")
```

Comment:
this should be the comment author and match what is in the sent comment username.